composer.json

```diff
@@ -20,12 +20,16 @@
     },
     "autoload": {
         "psr-0": {
-            "LightSaml\\Tests\\": "tests/",
             "LightSaml\\": "src/"
         }
     },
+    "autoload-dev": {
+        "psr-0": {
+            "LightSaml\\Tests\\": "tests/"
+        }
+    },
     "require": {
-        "php": ">=7.2.5",
+        "php": "^7.2.5|^8.0",
```

Comment:
I don't think this change is good. It will prevent the library from being used with PHP 9+.

Reply:
I don't understand what you say because the latest php version is 8.0.2 .

Reply:
I wanna exclude php 9+ and it's probably safe to exclude 8.1+, isn't it ?

Reply:
No it's not. Why would you want to artificially exclude future PHP versions? This will make it harder for users to experiment with them and report any bugs. See this thread by a core maintainer of Symfony: https://mobile.twitter.com/nicolasgrekas/status/1263023258938548225

Reply:
Sounds good to me :)